cg/store/models.py

```diff
@@ -999,3 +999,17 @@ class SampleRunMetrics(Base):
     __mapper_args__ = {
         "polymorphic_on": "type",
     }
+
+
+class IlluminaSampleRunMetrics(SampleRunMetrics):
```

Comment:
I think we should start adding doc strings since the database models will grow quite a bit
```suggestion
class IlluminaSampleRunMetrics(SampleRunMetrics):
"""Sequencing metrics for a sample sequenced on an Illumina instrument. The metrics are per sample, per lane, per flow cell."""
```